README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
 
 ### Supported GPUs
 
-#### Radeon Pro V620 and W6800 Workstation GPUs
+Support can be assumed for current server accelerators GPUs intended for enterprise customers.
+This ROCm release officially supports workstation GPU products.
+However, there are also compatible consumer GPU products available.
+Finally, this ROCm release can be used with a number of older GPU products.
 
-This release extends ROCm support for Radeon Pro V620 and W6800 Workstation GPUs.
+#### Enterprise GPUs
 
-- SRIOV virtualization support for Radeon Pro V620
-- KVM Hypervisor (1VF support only) on Ubuntu Host OS with Ubuntu, CentOs, and RHEL Guest
-- Support for ROCm-SMI in an SRIOV environment. For more details, refer to the ROCm SMI API documentation.
+Current enterprise GPU accelerators can be assumed to be supported. These include:
+
+- AMD Instinct™ MI100
+- AMD Instinct™ MI200
```

Comment:
This should probably be "AMD Instinct(tm) MI210". While we support the broad line of accelerators in the AMD Instinct MI200 series, none of them are actually called MI200 (that's really our name for the chip, not the product). The currently available products are MI210, MI250, and MI250X.

Reply:
Recently I have used https://llvm.org/docs/AMDGPUUsage.html#amdgpu-architecture-table trying to understand all these. But there are still a lot of TBA which should be updated with the real truth™. @jlgreathouse feel free to update the list there too if you have this precious knowledge. ;-)

Reply:
Thanks for the correction, @jlgreathouse! I also tried to figure out the elusive server product stack for enterprise customers using the LLVM wiki. Obviously, in this case at least, I was misled. :) I changed this line as suggested in my latest commit.